todo-app/src/components/todo/TodoItem.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div
+    class="flex flex-row sm:flex-col justify-between items-center"
+    @click="setIsEditingTrue"
+  >
+    <div class="flex flex-col">
+      <div class="flex flex-row sm:justify-between sm:w-38rem">
+        <TodoTitle :todo="todo" />
```

Comment:
Dont add the whole todo as prop, only the properties which needed in the component. Ex. just add title to the todoTitle component.